### PR TITLE
Header and footer

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Added Header and Footer Components to Ecommerce App. The rest of the pages will be added in future PRs.

## Files to be reviewed

- Header.jsx
- Footer.jsx
- footer.data.jsx
- Folder Structure

## Files to be ignored
- CSS files
- assets folder

### Please review this PR and give your feedback

### CODE REVIEW

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        <span className="offer-text border-right-1 pr-1 mr-1">
+          <i className="fas fa-truck px-1"></i>Free shipping on all orders
+        </span>
+        <span className="offer-code">Use code FREEDEL</span>
+      </div>
+      <header className="header flex-row wrap">
+        <div className="navbar-section">
+          <i className="fas fa-bars fa-lg primary-text myhamburger"></i>
+
+          <a href="/index.html" className="navbar-logo no-underline">
```
Comment (Maianki) [inline src/components/header/Header.jsx]: This ```a``` tag could also be converted to ```<Link>```.

**Review by Maianki — APPROVED** [review]
I found your code readable.

Some points that I liked -

The proper breaking up of large pieces of code into smaller reusable components.
The Use of destructuring at various places makes your code more readable and clean.

What you can work on: -
Formatting of code- I would recommend the use of any code format like prettier.

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      <div className="footer-item footer-links">
+        <h6 className="text-uppercase underline">Account</h6>
+        <ul className="links">
+          {accountLinks.map(({ id, linkName, url }, index) => (
+            <li key={id}>
+              <Link className="link" to={url}>
+                {linkName}
+              </Link>
+            </li>
+          ))}
```
Comment (BalajiHarinath) [inline src/components/footer/Footer.jsx:36]: Good that you have mapped the repeating structure and also de-structured the object. 
Suggestion : I think index is not required here.

**Review by BalajiHarinath — APPROVED** [review]

File: src/components/footer/Footer.jsx (hunk 2 above)
Comment (ApurvaSawant11) [reply src/components/footer/Footer.jsx:36]: Thanks! 
I'll need index later. That's why I have added it here.

File: src/components/header/Header.jsx (hunk 1 above)
Comment (ApurvaSawant11) [reply src/components/header/Header.jsx]: Ohh Missed it. Thanks for pointing it out. I have made the changes

**Pull request merged**